nexus_constructor/InstrumentView.py

```diff
@@ -47,17 +47,18 @@ def __init__(self, parent):
         self.num_neutrons = 9
 
         # Create lists for neutron-related objects
-        self.neutron_entities = []
-        self.neutron_meshes = []
-        self.neutron_transforms = []
+        self.neutron_objects = {
```

Comment:
stick a comment here that they're just stored because pyside2 qt3d has a bug that requires variables to be kept in scope